.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,72 @@
+name: Ubuntu Bionic / Linux
```

Comment:
The name of this file, `c-cpp.yml` could also be more descriptive, to differentiate well from other actions that may come to this folder.

Reply:
19e4896